<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaybackList</class>
 <widget class="QWidget" name="PlaybackList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTreeView
{
	font: 13px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_background" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="toolButton_close">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>75</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton
{
	border: 0px;
	border-image: url(:/playback/playback/return2.png);
	background-color: #FFFFFF;
}
QToolButton:hover
{
	background-color: rgb(10,168,227);
}
QToolButton:pressed
{
	background-color: #2E2E2E;
}
QToolButton:checked
{
	background-color: rgb(10,168,227);
}
QToolButton:disabled
{
	background-color: rgb(150,150,150);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_searchResult">
        <property name="text">
         <string>Search Result:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TreeView" name="treeView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_firstPage">
          <property name="styleSheet">
           <string notr="true">QToolButton
{
	border: 0px;
	border-image: url(:/playback/playback/startpage_w.png);
}
QToolButton:hover
{
	border-image: url(:/playback/playback/startpage_b.png);
}
QToolButton:pressed
{
	border-image: url(:/playback/playback/startpage_w.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_previousPage">
          <property name="styleSheet">
           <string notr="true">QToolButton
{
	border: 0px;
	border-image: url(:/playback/playback/prevpage_w.png);
}
QToolButton:hover
{
	border-image: url(:/playback/playback/prevpage_b.png);
}
QToolButton:pressed
{
	border-image: url(:/playback/playback/prevpage_w.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyLineEditPage" name="lineEdit_page">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly</set>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_nextPage">
          <property name="styleSheet">
           <string notr="true">QToolButton
{
	border: 0px;
	border-image: url(:/playback/playback/nextpage_w.png);
}
QToolButton:hover
{
	border-image: url(:/playback/playback/nextpage_b.png);
}
QToolButton:pressed
{
	border-image: url(:/playback/playback/nextpage_w.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_lastPage">
          <property name="styleSheet">
           <string notr="true">QToolButton
{
	border: 0px;
	border-image: url(:/playback/playback/endpage_w.png);
}
QToolButton:hover
{
	border-image: url(:/playback/playback/endpage_b.png);
}
QToolButton:pressed
{
	border-image: url(:/playback/playback/endpage_w.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PushButtonPage" name="pushButton_go">
          <property name="text">
           <string>Go</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_page">
          <property name="text">
           <string>Total: 500, Page: 1/11</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_time" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_prePlayback">
           <property name="text">
            <string>Pre Playback</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ComboBox" name="comboBox_prePlayback">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>5s</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10s</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>30s</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>60s</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>120s</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>300s</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>600s</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_postPlayback">
           <property name="text">
            <string>Post Playback</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="ComboBox" name="comboBox_postPlayback">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>5s</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10s</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>30s</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>60s</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>120s</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>300s</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>600s</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_button" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>35</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="MyPushButton" name="pushButton_export">
           <property name="text">
            <string>Export</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="MyPushButton" name="pushButton_exportAll">
           <property name="text">
            <string>Export All</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>36</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header>mypushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEditPage</class>
   <extends>QLineEdit</extends>
   <header>mylineeditpage.h</header>
  </customwidget>
  <customwidget>
   <class>TreeView</class>
   <extends>QTreeView</extends>
   <header>treeview.h</header>
  </customwidget>
  <customwidget>
   <class>PushButtonPage</class>
   <extends>QPushButton</extends>
   <header>PushButtonPage.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
